### DISCUSSION TITLE
Custom MacOS folder Icons

### DISCUSSION THREAD

**Discussion by JamesMcMahon** [samschott/maestral#737] [Ideas]
Hey all,

I generated some custom folder icons from the Maestral icon using [folderify](https://github.com/lgarron/folderify). Thought folks might enjoy these.

Curious if there was any interest in incorporation these officially in the project?

## Dark
![icon_512x512@2x](https://user-images.githubusercontent.com/916676/197426143-b36831e5-90de-4516-a4d6-7105350230e3.png)

## Light
![icon_512x512@2x](https://user-images.githubusercontent.com/916676/197426151-98470ef7-a892-4be9-b52c-c3fbf3a521e4.png)

**Comment by robault** [comment]
I wasn't sure how to do this, but a simple drag and drop into the Get Info window and it's customized. Very cool, thank you!

https://support.apple.com/guide/mac-help/change-icons-for-files-or-folders-on-mac-mchlp2313/mac

**Reply by JamesMcMahon** [reply]
My pleasure

**Comment by samschott** [comment]
They look really nice! I'll see if those can be incorporated. My main hesitancy comes from needing to maintain and update this as macOS folder icons may change in the future.
